static/i18next-parser.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Configuration for i18next-parser. See https://github.com/i18next/i18next-parser
+ * NOTE: run i18next-parser from the root of the repo
+ */
+module.exports = {
+  locales: ["en-US"],
```

Comment:
if this only exports en-US, where did the es/place.json come from?

Reply:
I see, I think you handwrote the files? I ran the code and the namespace was translation.json not place.json.